docs/contributing/Build-XKit.md

```diff
@@ -81,41 +77,23 @@ Lints CSS files using CSSLint and reports the output.
 
 Top-level build task.
 
-See also: [`gulp build:chrome`](#gulp-buildchrome), [`gulp build:firefox`](#gulp-buildfirefox), [`gulp build:safari`](#gulp-buildsafari).
-
-#### `gulp build:chrome`
-
-Builds the Chrome browser extension from source and outputs it to `build/chrome/`.  Also creates an archive, `new-xkit-x.x.x.zip`, and outputs it to `build/chrome/`.
-
-#### `gulp copy:chrome`
-
-Builds the Chrome browser extension from source and outputs it to `build/chrome/`.
-
-See also: [`gulp build`](#gulp-build), [`gulp build:chrome`](#gulp-buildchrome), [`gulp compress:chrome`](#gulp-compresschrome).
-
-#### `gulp compress:chrome`
-
-Creates an archive, `new-xkit-x.x.x.zip`, from an existing Chrome source build and outputs it to `build/chrome/`.
-
-See also: [`gulp build`](#gulp-build), [`gulp build:chrome`](#gulp-buildchrome), [`gulp copy:chrome`](#gulp-copychrome).
-
-#### `gulp build:firefox`
+See also: [`gulp build:webext`](#gulp-buildwebext), [`gulp build:safari`](#gulp-buildsafari).
 
-Builds the Firefox browser extension from source and outputs it to `build/firefox/`.  Also creates a [Cross-platform Installer Module](https://developer.mozilla.org/en/docs/XPI), `@new-xkit-x.x.x.xpi`, and outputs it to `build/firefox/`.
+#### `gulp build:webext`
 
-See also: [`gulp compress:firefox`](#gulp-compressfirefox), [`gulp copy:firefox`](#gulp-copyfirefox).
+Builds the Chrome/Firefox browser extension from source and outputs it to `build/webext/`.  Also creates an archive, `new-xkit-x.x.x.zip`, and outputs it to `build/webext/`.
 
-#### `gulp copy:firefox`
+#### `gulp copy:webext`
 
-Builds the Firefox browser extension from source and outputs it to `build/firefox/`.
+Builds the Chrome/Firefox browser extension from source and outputs it to `build/webext/`.
 
-See also: [`gulp build`](#gulp-build), [`gulp build:firefox`](#gulp-buildfirefox), [`gulp compress:firefox`](#gulp-compressfirefox).
+See also: [`gulp build`](#gulp-build), [`gulp build:webext`](#gulp-buildwebext), [`gulp compress:webext`](#gulp-compresswebext).
 
-#### `gulp compress:firefox`
+#### `gulp compress:webext`
 
-Creates a [Cross-platform Installer Module](https://developer.mozilla.org/en/docs/XPI), `@new-xkit-x.x.x.xpi`, from an existing Firefox source build and outputs it to `build/firefox/`.
+Creates an archive, `new-xkit-x.x.x.zip`, from an existing Chrome/Firefox source build and outputs it to `build/webext/`.
```

Comment:
Should have a note that during the release process, this archive is uploaded directly to AMO for signing, which returns a signed XPI that we host on Github.